examples/apps/tree-cli-app/data/default.verbose.json

```diff
@@ -0,0 +1 @@
+{"type":"com.fluidframework.example.cli.List","fields":[]}
```

Comment:
is the verbose version supposed to not have any fields?

Reply:
the default tree does not have any children under this node, so yes. I have however changed the default tree to be a bit bigger so its a better example.